From 357c2f88aef9fe7a500e21d0faf490f2bdb4b002 Mon Sep 17 00:00:00 2001
From: Yusuke Goda <yusuke.goda.sx@renesas.com>
Date: Thu, 30 Nov 2017 18:47:26 +0900
Subject: [PATCH 667/909] arm64: dts: h3ulcb: Add cpu-supply property in a57_0
 node

commit 727c8ba03bf3c2aefdd700858db961721025001e from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This adds cpu-supply property in a57_0 node

Signed-off-by: Yusuke Goda <yusuke.goda.sx@renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/ulcb.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/ulcb.dtsi b/arch/arm64/boot/dts/renesas/ulcb.dtsi
index b2c0d2df643f..ce1ce5f27a40 100644
--- a/arch/arm64/boot/dts/renesas/ulcb.dtsi
+++ b/arch/arm64/boot/dts/renesas/ulcb.dtsi
@@ -156,6 +156,10 @@
 	};
 };
 
+&a57_0 {
+	cpu-supply = <&dvfs>;
+};
+
 &du {
 	status = "okay";
 };
-- 
2.17.1

